src/main/java/sirius/db/text/PatternExtractProcessor.java

```diff
@@ -19,7 +21,9 @@
  */
 public class PatternExtractProcessor extends ChainableTokenProcessor {
 
-    private static final Pattern EXTRACT_EMAILS = Pattern.compile("(\\p{Alnum}.+)@(.+)$");
+    @SuppressWarnings("java:S5852")
+    @Explain("This regex cannot cause catastrophic backtracking.")
```

Comment:
Is it possible to fix this with possessive quantifiers, rather than suppressing the warning?

Reply:
We just had a production issue, which might correlate to this regex - maybe we revise again...